app/components/mu-login.hbs

```diff
@@ -4,19 +4,19 @@
       {{t "utility.username"}}&nbsp;
     </AuLabel>
     <AuInput
-      type="text"
+      @type="text"
       id="nickname"
-      {{on "input" (fn this.setNickname this.nickname)}}
+      @value={{this.nickname}}
     />
   </div>
   <div>
     <AuLabel for="password">
       {{t "utility.password"}}&nbsp;
     </AuLabel>
     <AuInput
-      type="password"
+      @type="password"
       id="password"
-      {{on "input" (fn this.setPassword this.password)}}
+      @value={{this.password}}
```

Comment:
small note: I think you don't want to change this to only a @value hook, as this is a double-bind which might disappear (or get deprecated) in the future: https://github.com/appuniversum/ember-appuniversum/issues/250

so instead `{{on 'input' this.setPassword}}` and in class: `setPassword(event) {this.password = event.target.value;}`

Reply:
Ok makes sense, I'll make the required changes.